File: med-app-backend-spring/src/main/java/com/medapp/app/dts/medappbackendspring/Dto/DoctorProfileInfoForClient.java
package com.medapp.app.dts.medappbackendspring.Dto;

import com.medapp.app.dts.medappbackendspring.Entity.Category;
import com.medapp.app.dts.medappbackendspring.Entity.Feedback;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class DoctorProfileInfoForClient {
    private Long id;
    private String firstname;
    private String lastname;
    private String city;
    private String email;
    private String description;
    private Long phoneNumber;
    private List<FeedbackDto> receivedFeedbacks;

}

File: med-app-android/app/src/main/java/dts/app/med_app_android/Model/DoctorProfileInfoForClient.kt
package dts.app.med_app_android.Model

data class  DoctorProfileInfoForClient(
    val description: String,
    val firstname: String,
    val city: String,
    val id: Int,
    val lastname: String,
    val email: String,
    val phoneNumber: Long,
    val receivedFeedbacks: List<FeedbackDto>
)